src/SFA.DAS.LevyTransferMatching.Web/Models/Cache/CreateApplicationCacheItem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Cache
+{
+    public class CreateApplicationCacheItem
+    {
+        private IEnumerable<string> _additionalEmailAddresses;
```

Comment:
OK:

* This has been replaced for a regular `List<string>` creation in the constructor;
* As discussed previously today, this is for rendering purposes, dealing with this in the Orchestrator, rather than the view itself to keep the view clear of bloat